core/src/test/scala/org/apache/spark/scheduler/EventLoggingListenerSuite.scala

```diff
@@ -75,6 +75,10 @@ class EventLoggingListenerSuite extends SparkFunSuite with LocalSparkContext wit
     testApplicationEventLogging()
   }
 
+  test("End-to-end event logging with exit code") {
+    testEventLogging(exitCode = Some(0))
```

Comment:
Forgoit to mention it, can you add another case here where `exitCode = None` ?

Reply:
sure, i can add an explicit test, but since default is None, all other tests basically test exitCode = None